source/GpioController.cs

```diff
@@ -11,7 +11,7 @@ namespace Windows.Devices.Gpio
     /// Represents the default general-purpose I/O (GPIO) controller for the system.
     /// </summary>
     /// <remarks>To get a <see cref="Gpio​Controller"/> object, use the <see cref="GetDefault"/> method.</remarks>
-    public sealed class Gpio​Controller
+    public sealed class Gpio​Controller : IGpioController
```

Comment:
These changes probably require that we now have to provide implementation for `GetControllers(IGpioProvider provider)`...

Or not, if we choose not to provide _discoverability_ for these controllers. 🤔 

I mean, we are after saving flash and ram space here, and if one has implemented it's own GPIO controller it's logical that he's has to be aware of that controller, right?!

Reply:
I don't think the GetControllers will need to be implemented.  The reason for making the IGpioController and IGpioPin interfaces public was really to avoid having to implement the Provider model.  

Even under UWP they are not used for discovery.

Reply:
Good! So we are in agreement that this is not needed.